.git-blame-ignore-revs

```diff
@@ -1,3 +1,5 @@
 # Conversion to new code style
 45b9887603f08523f9a663b164bd9f10b85f8b25
 9114d609168b327f00ef47b9891733cdb119b512
+# Change of code style
```

Comment:
I don't think we need the extra comment. Whenever we change the style, whatever we change to becomes the "new style", so the previous comment is just as applicable. If anything you could edit the first comment to read "Change of code style" and remove this one.

Reply:
Well, the 1st 2 commits where the initial setup, the last is a change of that, further changes would just get appended, no new headline needed